google/cloud/storage/client.h

```diff
@@ -68,6 +68,32 @@ struct ClientImplDetails;
  * For example, access tokens need to be refreshed from time to time, and this
  * may impact the performance of some operations.
  *
+ * @par Connection Pool
+ * By default this class uses HTTPS to communicate with the service. Creating a
+ * new HTTPS session is relatively expensive, as it must go through the TCP/IP
+ * and SSL handshakes. To minimize this overhead the class maintains a
+ * connection pool to the service. After each request completes the connection
+ * is returned to the pool, and reuse in future requests. Note that for
+ * downloads (implemented by the ReadObject() member function) the connection
+ * remains in use until the download completes. Therefore, having multiple
+ * downloads open at the same time requires multiple connections.
+ *
+ * The application can limit the maximum size of this connection pool using
+ * `storage::ConnectionPoolSizeOption`. If returning a connection to the pool
+ * would make the pool larger than this limit then the oldest connection in the
```

Comment:
nit: Would "oldest inactive connection" be more accurate?

The paragraph below suggests that an active download will not be closed.

Your call on whether to update the text or not.

Reply:
All connections in the pool are inactive ... and yes, active downloads are not closed (not while in progress).

Reply:
hmm ok.

s/connecitons in the pool are inactivate/connections in the pool are inactive/

Reply:
Fixed.